src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.scss

```diff
@@ -117,17 +117,9 @@ $whitecol: white;
   .edit {
     @include btnstyle;
 
-    color: var(--ubs-electric-violet);
-    border-color: var(--ubs-electric-violet);
-    margin-left: 0;
-  }
-
-  .edit:hover {
-    background-color: $whitecol;
-  }
-
-  .edit:not(:disabled):not(.disabled):active {
-    background-color: $whitecol;
+    background-color: var(--ubs-button-light-green);
+    color: #1d2830;
```

Comment:
in line 121 background color should be declared also as a variable (in file typography/_colorsUbs.scss it's "--ubs-quaternary-dark-grey")